package/yast2-samba-server.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Mar 23 17:55:31 UTC 2016 - lmuelle@suse.com
+
+- Don't require libsmbclient as it's pulled in by libsmbclient-devel;
```

Comment:
A semicolon is commonly used as a separator between changelog descriptions and bug numbers.